ReleaseChecklist.md

```diff
@@ -32,6 +32,30 @@
 ### Homebrew and MacOS
  - [ ] Update the version and the hash (``sha256sum solidity_x.x.x.tar.gz``) in https://github.com/ethereum/homebrew-ethereum/blob/master/solidity.rb
  - [ ] Take the binary from the ``b_osx`` run of the released commit in circle-ci and add it to the release page as ``solc-macos``.
+ - [ ] Bump homebrew formula
+
+#### Bump homebrew formula using `brew`
+```
+brew bump-formula-pr --message="Update solidity to x.y.z" --url="https://github.com/ethereum/solidity/releases/download/vx.y.z/solidity_x.y.z.tar.gz" --sha256="<SHA256-HASH>" solidity
+```
+
+#### Bump homebrew formula manually
+
+- fork homebrew core repository `https://github.com/Homebrew/homebrew-core`
```

Comment:
This is too much for the checklist. Isn't it almost exactly the same as in https://github.com/ethereum/solidity/pull/9734/files#diff-a64159d53d82ffc076743ededd94db5eR33 ? This CAN be done via the github web interface, can it?

Reply:
:D yes, it is basically exactly the same as described there, somehow I missed that.. sure yes, it can be done via the github web interface.. i will simplify that